src/Renderer/LayeredMaterial.js

```diff
@@ -491,5 +493,9 @@ LayeredMaterial.prototype.setSelected = function setSelected(selected) {
     this.uniforms.selected.value = selected;
 };
 
+LayeredMaterial.prototype.setLayerZFactor = function setLayerZFactor(zFactor) {
```

Comment:
Is this method useful? It's just a one-line method and you don't use it https://github.com/iTowns/itowns/pull/511/files#diff-caf5227be4f9e2f283d82df32a678e4bL523 so I'd suggest removing it for now.

Reply:
yes, i used it in updateLayeredMaterialNodeElevation to deform geoid

Reply:
I meant: it's a one-liner and you only use 1 once so I'd prefer you to remove it (and replace it's usage in `updateLayeredMaterialNodeElevation` with `material.uniforms.zFactor.value = layer.zFactor;`)

Reply:
done